### ISSUE TITLE
CDC timestamp fields should be convert to timestamp

### ISSUE THREAD

**Issue by marcosmarxm** [airbytehq/airbyte#4159]
## Expected Behavior
They should be convert to timestamp

## Current Behavior
They are saved as long. See example
![image](https://user-images.githubusercontent.com/5154322/122236210-b578b300-ce94-11eb-9f7c-f52c47af3051.png)

## Logs
*If applicable, please upload the logs from the failing operation. For sync jobs, you can download the full logs from the UI by going to the sync attempt page and clicking the download logs button at the top right of the logs display window.*

## Steps to Reproduce
1.
1.
1.

## Severity of the bug for you
Very low / Low / **Medium** / High / Critical

## Airbyte Version
Found in the .env file in the root of the project.

## Connector Version (if applicable)
Found in the admin page in the UI in the respective Source or Destination tab.

## Additional context
This happens for Postgres and Mysql
https://github.com/airbytehq/airbyte/blob/85ee72d12f93a2159d26d4e490498e1d301d620c/airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/DebeziumEventUtils.java#L64

**Comment by Phlair** [comment]
Also happens for MSSQL (https://github.com/airbytehq/airbyte/issues/4542)

**Comment by danieldiamond** [comment]
and snowflake. this one would be seriously valuable to us

**Comment by danieldiamond** [comment]
This PR does not seem to do anything https://github.com/airbytehq/airbyte/pull/5356
or rather the bug issue was not properly articulated.

There are columns in a source table that are timestamp and when using CDC to load the data, these columns are converted to epoch time.

See additional notes: https://github.com/airbytehq/airbyte/pull/5234#issuecomment-894016478

tl;dr: The destination table should have same schema regardless of whether a user chooses CDC or standard loading.

**Comment by subodh1810** [comment]
@danieldiamond I guess there is some confusion around the aim of the PR and this issue. 
Base on the PR that you rasied https://github.com/airbytehq/airbyte/pull/5234 , I thought you wanted to change the type of the column `_ab_cdc_updated_at` specifically. It was previously sent as a number type (long value) and with your approach it  was changed to a TIMESTAMP type. It works as expected. 

Regarding the EPOCH value of the column `_ab_cdc_updated_at`, let me explain what the column represents. It indicates the time that the change was made in the database. This is extracted from the logs during replication (bin log in case of mysql). But when you run the connector for the first time, we dont directly do log based replication. We first do a `SELECT *` query on the table to get the pre existing data and once thats complete then we start reading the logs for CDC. In the `SELECT * ` approach, we dont have access to the information when a specific change was made and for that reason the value `_ab_cdc_updated_at` is set to EPOCH. Also we use debeizum to handle all of this for us.

My entire understanding of what you wanted was based on the PR that you raised. 

I think what you are specifically looking for is this https://github.com/airbytehq/airbyte/issues/5376 for which I have another PR https://github.com/airbytehq/airbyte/pull/5389 which would try to tackle `The destination table should have same schema regardless of whether a user chooses CDC or standard loading.`

**Comment by danieldiamond** [comment]
yeah i was worried after I reread the description of the PR that this might be the case. and that other PR https://github.com/airbytehq/airbyte/pull/5389 is 💯 what I had intended! Glad it's still being pursued and thanks again for all your help
